dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+PROJECT_NAME=zammad-dev
+COMPOSE_FILE=".dev/docker-compose.yml"
+ENV_FILE=".dev/.env"
+ENV_TEMPLATE=".dev/.env.dist"
+
+# Ensure .env exists
+if [ ! -f "$ENV_FILE" ]; then
+  echo "📝 .env file not found, copying default from .env.dist"
+  cp "$ENV_TEMPLATE" "$ENV_FILE"
+fi
+
+echo "🧹 Removing previous containers..."
+docker-compose -f "$COMPOSE_FILE" down --volumes --remove-orphans || true
+
+echo "🔧 Building and starting containers..."
+COMMIT_SHA=$(git rev-parse HEAD) RAILS_ENV=development docker-compose -f "$COMPOSE_FILE" up -d --remove-orphans
+
+echo "⚙️  Running one-shot DB initializer..."
+docker compose -f "$COMPOSE_FILE" run --rm zammad zammad-init
```

Comment:
This does run the DB init for me, but then hangs with `zammad-init waiting for elasticsearch server to be ready…` repeating forever - maybe because of wrong hostname (see .env)?

Reply:
Colleages have signalled interest in this. There are already some private solutions, so your PR could finally be a starting point for an official, shared solution that we can improve together, step by step.